src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOUR_INDEX = 2;
+    private static final int INCOME_INDEX = 3;
+    private static final String HEADER = "Report for period ";
+    private static final String SEPARATOR = " - ";
+    private static final String DELIMITER = " ";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate endDate = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder result = new StringBuilder();
+        result.append(HEADER)
```

Comment:
chain your append calls like that
```suggestion
        StringBuilder result = new StringBuilder()
                                          .append(HEADER)
                                          .append(...
```